laser_encoders/download_models.py

```diff
@@ -45,7 +45,11 @@ def __init__(self, model_dir: str = None):
         self.base_url = "https://dl.fbaipublicfiles.com/nllb/laser"
 
     def download(self, filename: str):
-        url = os.path.join(self.base_url, filename)
+        # Because on windows os.path.join will use "\" insted of "/", so link would be:
+        # https://dl.fbaipublicfiles.com/nllb/laser\laser2.pt instead of https://dl.fbaipublicfiles.com/nllb/laser/laser2.pt
+        # which results in a failed download.
+        url = f"{self.base_url}/{filename}"
+        #url = os.path.join(self.base_url, filename)
```

Comment:
Can you please remove this line that you have commended out?

Reply:
Removed.